index.js

```diff
@@ -19,6 +19,8 @@ module.exports = {
       'silvermine/call-indentation': 'error',
       'silvermine/fluent-chaining': 'error',
       'silvermine/no-multiple-inline-functions': 'error',
+      'silvermine/no-multiline-var-declarations': 'error',
```

Comment:
This plugin name is defined in https://github.com/silvermine/eslint-plugin-silvermine/pull/4/ as `silvermine/no-multiline-var-declaration` (singular). The name should either be fixed here or in that PR.